sdk/resourcemanager/azure-resourcemanager-samples/src/test/java/com/azure/resourcemanager/samples/AppServiceSampleLiveOnlyTests.java

```diff
@@ -136,8 +142,11 @@ public void testManageFunctionAppLogs() throws IOException {
         if (skipInPlayback()) {
             return;
         }
-        // require a longer read timeout in HttpClient
-        // e.g., new NettyAsyncHttpClientBuilder().readTimeout(Duration.ofMinutes(10))
+        azureResourceManager = AzureResourceManager.authenticate(
+            setReadTimeout(azureResourceManager.storageAccounts().manager().httpPipeline(), Duration.ofMinutes(10)),
+            new AzureProfile(azureResourceManager.tenantId(), azureResourceManager.subscriptionId(), AzureEnvironment.AZURE)
+        )
+            .withDefaultSubscription();
```

Comment:
would need to change `authenticate(httpPipeline, profile)` in https://github.com/Azure/azure-sdk-for-java/pull/16144